pkg/skuba/actions/cluster/upgrade/plan.go

```diff
@@ -78,5 +76,19 @@ func Plan() error {
 		}
 	}
 
+	// fetch addon upgrades for the next available cluster version
+	nextClusterVersion := upgradePath[0]
```

Comment:
Does it make sense to return an array on line#51 if you are defaulting to just the first option?

Reply:
yes it's fine, we need the cluster upgrade paths in an array format, to be able to show 2 step upgrades as well (e.g. 1.14.3 -> 1.15.0 -> 1.16.0)